<launch>

  <!-- Arguments -->
  <arg name="bag" doc="absolute path to bag to be replayed"/>
  <arg name="topics_to_connect" doc="images pair topic to connect"/>

  <!-- Nodes execution -->
  <node pkg="rosbag" type="play" name="player" output="screen" args="$(arg bag) "/>

  <node pkg="infuse_debug_tools" name="asn1_bitstream_to_disparity_image" type="asn1_bitstream_to_disparity_image_node" output="screen">
    <param name="topics_to_connect" value="$(arg topics_to_connect)" />
  </node>

</launch>
